core/templates/dev/head/pages/exploration_player/LearnerViewInfo.js

```diff
@@ -114,6 +116,8 @@ oppia.controller('LearnerViewInfo', [
               expInfo.last_updated_msec);
             $scope.numViews = expInfo.num_views;
             $scope.objective = expInfo.objective;
+            $scope.isExplorationPrivate = (
```

Comment:
explorationIsPrivate (so that it is clearer that it's a bool, as opposed to a function).

Reply:
okay.